packages/react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -381,6 +381,13 @@ export interface ComboBoxProps<ItemType>
    * Provide the text that is displayed when the control is in warning state
    */
   warnText?: ReactNode;
+
+  /**
+   * Props to be passed directly to the underlying <input> element.
+   * Useful for customizing input behavior, such as setting maxLength, placeholder,
+   * Note: These props are merged with internal input props. Custom values will override defaults when applicable.
+   */
+  inputProps?: React.InputHTMLAttributes<HTMLInputElement>;
```

Comment:
`InputHTMLAttributes` is already imported in this file.

```suggestion
  inputProps?: InputHTMLAttributes<HTMLInputElement>;
```